samples/benchmarks/coremark/sysbuild.cmake

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+if(${images_added})
+  # same source directory is used for all targets
+  # to avoid recursive calls do not add project if remote was already added
+  return()
+endif()
```

Comment:
This set and lock behaviour seems unclean. @nordicjm, any recommendations on how to make this function run only for the main application image?

We use sysbuild here to have the same application image (Coremark sample) built for the configured CPUs on a given SoC